static/src/javascripts/projects/common/modules/ui/dropdowns.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import bean from 'bean';
+import fastdom from 'fastdom';
+
+const containerSelector = '.dropdown';
+const buttonCN = 'dropdown__button';
+const contentCN = 'dropdown__content';
+
+const updateAria = (container: Element): void => {
+    const v: boolean = container.classList.contains('dropdown--active');
+    const content = [...container.getElementsByClassName(contentCN)];
+    const button = container.getElementsByClassName(buttonCN);
+
+    content.forEach((c: Element) => {
+        c.setAttribute('aria-hidden', v ? 'false' : 'true');
```

Comment:
You could do `.setAttribute('aria-hidden', !v)` and remove the conditional ternary operator here as the the value of `!v` will be converted to a string by `setAttribute`.